### PR TITLE
Clear focus on font size EditText when font size SeekBar progress changed

### LINKED ISSUE #315: Host creation: font size slider value does not update when focused
When the user is focused on the value of the font size, dragging the slider does not change the value. 

GIF included:
output

### PR DESCRIPTION
**Pull request by johnheroy**
Add 
```
android:focusable="true"
android:focusableInTouchMode="true"
```
to the HostEditorFragment LinearLayout so focus doesn't jump back to the quick connect EditText when clearing focus on the font size EditText. In order though that we preserve a auto-focused quick connect container after tapping '+', manually request focus in the fragment's onCreate method.

Fixes #315.

### CODE REVIEW

File: app/src/main/res/layout/fragment_host_editor.xml [hunk 1]
```diff
@@ -25,6 +25,8 @@
 	>
 
 	<LinearLayout
+		android:focusable="true"
```
Comment (khorimoto) [inline app/src/main/res/layout/fragment_host_editor.xml]: Why do we need these changes?

Comment (johnheroy) [inline app/src/main/res/layout/fragment_host_editor.xml]: Without these changes, when we clear focus on the font size EditText, focus automatically returns to the first focusable element, which is the quick connect container. It's just kind of janky UX to be fiddling with the font size and then have focus jump to a different section in the fragment.

Comment (khorimoto) [inline app/src/main/res/layout/fragment_host_editor.xml]: As discussed offline, LGTM, but please add comments explaining why these are necessary. Thanks!

Comment (johnheroy) [inline app/src/main/res/layout/fragment_host_editor.xml]: Done.

**Pull request merged**